src/quo2/screens/channel_avatar.cljs

```diff
@@ -0,0 +1,51 @@
+(ns quo2.screens.channel-avatar
+  (:require [reagent.core :as reagent]
+            [quo.react-native :as rn]
+            [quo.previews.preview :as preview]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.channel-avatar :as quo2]))
+
+(def descriptor [{:label "Big?"
+                  :key   :big?
+                  :type  :boolean}
+                 {:label "Dark"
+                  :key   :dark?
+                  :type  :boolean}
+                 {:label "Avatar color"
+                  :key   :avatar-color
+                  :type  :text}
+                 {:label "Avatar color"
+                  :key   :avatar-text
+                  :type  :text}
+                 {:label   "Lock status"
+                  :key     :lock-status
+                  :type    :select
+                  :options [{:key   :unlocked
+                             :value "Unlocked"}
+                            {:key   :locked
+                             :value "Locked"}]}])
```

Comment:
should we also add `:none` as available option?

Reply:
I think there's a clear option on the `select` on the preview screen